src/main/java/com/ericsson/gerrit/plugins/eiffel/handlers/SomeRuntimeException.java

```diff
@@ -0,0 +1,8 @@
+package com.ericsson.gerrit.plugins.eiffel.handlers;
+
+public class SomeRuntimeException extends Exception {
```

Comment:
An exception is not a runtime exception. From https://docs.oracle.com/javase/8/docs/api/?java/lang/Throwable.html

> The class `Exception` and any subclasses that are not also subclasses of `RuntimeException` are checked exceptions.

Reply:
Exception name changed